workflow/kfdrc_production_somatic_variant_cnv_wf.cwl

```diff
@@ -475,6 +475,7 @@ steps:
       input_tumor_name: input_tumor_name
       input_normal_aligned: input_normal_aligned
       input_normal_name: input_normal_name
+      manta_small_indels: run_manta/manta_small_indels
```

Comment:
hmmm, might consider making this optional as we haven't actually committed to making this our standard SOP. Need to run by @yuankunzhu  first...

Reply:
let's make this optional.

Reply:
Ok I added a new variable that allows the value to be toggled. Had to rewrite some other stuff to fit it in. New tests for strelka below. Let me know if I missed any tests.